### DISCUSSION TITLE
Creating reports in specific language using parameter.

### DISCUSSION THREAD

**Discussion by yashnalawade11** [amzn/selling-partner-api-models#3619] [General] [answered]
I need to create a Selling Partner report in 'English' language. However, the default language for report data is Spanish.
The report type is 'GET_MERCHANT_LISTINGS_ALL_DATA'.
Is there any way to create the report in the required language?

**Comment by rugved1991** [comment, marked as answer]
Hi @yashnalawade11,

The report language preference is determined by the seller in their account settings. As a developer, it is not possible to request a report in your preferred language through an API. You will have to work with the seller to change the language preference of the reports in their seller account.
Please check out this GitHub issue discussion for more information: https://github.com/amzn/selling-partner-api-docs/issues/198

Best,
Rugved
Solutions Architect, SP API
